applications/mne_analyze/plugins/annotationmanager/annotationmanager.cpp

```diff
@@ -171,15 +168,18 @@ QWidget *AnnotationManager::getView()
 void AnnotationManager::handleEvent(QSharedPointer<Event> e)
 {
     switch (e->getType()) {
-        case EVENT_TYPE::NEW_ANNOTATION_ADDED:
-            emit newAnnotationAvailable(e->getData().toInt());
-            onTriggerRedraw();
-            break;
-        case EVENT_TYPE::SELECTED_MODEL_CHANGED:
-            onModelChanged(e->getData().value<QSharedPointer<ANSHAREDLIB::AbstractModel> >());
-            break;
-        default:
-            qWarning() << "[AnnotationManager::handleEvent] Received an Event that is not handled by switch cases.";
+    case EVENT_TYPE::NEW_ANNOTATION_ADDED:
+        emit newAnnotationAvailable(e->getData().toInt());
+        onTriggerRedraw();
+        break;
+    case EVENT_TYPE::SELECTED_MODEL_CHANGED:
+        onModelChanged(e->getData().value<QSharedPointer<ANSHAREDLIB::AbstractModel> >());
+        break;
+    case MODEL_REMOVED:
```

Comment:
why don't you use `EVENT_TYPE::` here as well? Check other files as well.

Reply:
It seems we don't need to use the enum name before the event names. Should we standardize to use them always for clarity, or just get rid of them?